ui/packages/catalog-ui-search/src/main/webapp/component/query-interactions/query-interactions.view.js

```diff
@@ -35,7 +36,94 @@ const createDuplicateQuery = attributes => {
 }
 
 module.exports = Marionette.ItemView.extend({
-  template: template,
+  template() {
+    const disabledExport = store.getSelectedResults().length < 1
+    const exportClass = disabledExport ? 'composed-menu' : ''
+    return (
+      <React.Fragment>
+        <div
+          className="query-interaction interaction-run"
+          title="Run"
+          data-help="Executes the search."
+        >
+          <div className="interaction-text">Run</div>
+        </div>
+        <div
+          className="query-interaction interaction-stop"
+          title="Stop"
+          data-help="Stops a search in the progress of executing.  If results have already returned from some sources, you won't lose them."
+        >
+          <div className="interaction-text">Stop</div>
+        </div>
+        <div
+          className="query-interaction interaction-delete"
+          title="Delete"
+          data-help="Deletes the search."
+        >
+          <div className="interaction-text">Delete</div>
+        </div>
+        <div
+          className="query-interaction interaction-duplicate"
+          title="Duplicate"
+          data-help="Create a new search based off this one."
+        >
+          <div className="interaction-text">Duplicate</div>
+        </div>
+        <div className="is-divider composed-menu" />
+        <div
+          className="query-interaction interaction-refresh-result-count"
+          title="Refresh Result Count"
+          data-help="Executes the search, returning only the result count."
+        >
+          <div className="interaction-text">Refresh Result Count</div>
+        </div>
+        <div className="is-divider composed-menu" />
+        <div className="interaction-versioning composed-menu">
+          <div
+            className="query-interaction interaction-deleted interaction-search-archived"
+            title="Search Archived"
+            data-help="Executes the search, but specifically looks for archived results."
+          >
+            <div className="interaction-text">Search Archived</div>
+          </div>
+          <div
+            className="query-interaction interaction-historic interaction-search-historical"
+            title="Search Historical"
+            data-help="Executes the search, but specifically looks for historical data."
+          >
+            <div className="interaction-text">Search Historical</div>
+          </div>
+        </div>
+        <div className="is-divider composed-menu" />
+        <div
+          className="query-interaction interaction-annotations"
+          title="View Notes"
+          data-help="View notes on the search."
+        >
+          <div className="interaction-text">View Notes</div>
+        </div>
+        <div
+          className="query-interaction interaction-feedback"
+          title="Submit Feedback"
+          data-help="Brings up a form to submit comments about your current search and it's results."
```

Comment:
✏️ personal again, I just don't like the word "you" in software 😂 
`...about the current search...`

✏️ *its

Reply:
I will fix the typo however this isn't my wording so I'm going to leave it the same